### DISCUSSION TITLE
這個APP有病毒!

### DISCUSSION THREAD

**Discussion by 11timyyy** [1101-GS4719-C/Python-Community-GS4719-C#424] [General]
在Google Play 商店中，有一款名為「Antivirus, Super Cleaner」的應用程式，這個看似無害的APP，還寫著"antivirus"，讓人不禁放下戒心。但是，他其實就是個"virus"。
這個應用程式會透過「SharkBot」新一代 Android 銀行木馬病毒，來竊取以及入侵用戶的手機，甚至入侵銀行帳戶把錢給轉走。
文章中提到，這個病毒和另一款 Android 病毒 Teabot運作原理不同，Teabot 是一種藏在 QR Code 應用程式之中的病毒。
Android的使用者們要多留心不要誤下載了!

文章來源:https://technews.tw/2022/03/08/android-antivirus-super-cleaner/

**Comment by ghost** [comment]
感謝分享，日後會注意，是說還是別輕易下在任何清理垃圾快取之類的app比較好

**Comment by pekoranoinu** [comment]
感謝你的分享，這提醒了我們不能完全相信google play商店。

**Comment by cordydeng** [comment]
感謝原po的提醒，順便在這裡補充一下，除了「Antivirus, Super Cleaner」以外，還有「Powerful Cleaner」、「Atom Clean-Booster」及「Alpha Antivirus , Cleaner」等app有相同的木馬軟體

**Reply by 11timyyy** [reply]
太讚了感謝補充!😁

**Comment by chupochun** [comment]
謝謝提醒 原來即使是在google play上的程式也不是完全安全的

**Comment by Yuhsin000** [comment]
他怎麼還沒下架QQ，怕

**Comment by bloodtrail0511** [comment]
太震驚了!病毒居然是偽裝在防毒軟體裡的，看來Google Play的審核還有待加強。現在就算看到正當渠道的軟體也不能隨便下載了

**Comment by ThomasMeng0515** [comment]
我也很好奇竟然大家都回報它是病毒了怎麼沒被下架...
